cmd/config.go

```diff
@@ -199,8 +199,14 @@ func getAccount() (*account, error) {
 		Secret:   "",
 	}
 
+	isRestricted := false
+
+	if askQuestion("is it a restricted account?") {
```

Comment:
What's a "restricted account"?

Reply:
It is an account whose API key has been created with IAM and which is of the "restricted" type

Reply:
We can avoid the question with something like this https://github.com/exoscale/cli/compare/iam/ch7024...7fELF/iam/ch7024

Reply:
Thank you @7fELF, I was about give a similar answer: ideally, we'd even identify the error at egoscale level and return an `egoscale.ErrorPermissionDenied` error that can be matched at client level and return an appropriate error message without asking questions...

Reply:
The fact of asking the question is not the problem, the problem was to find the best way to solve the bug without removing, as it was your idea, to remove code that for me seems useful.
And yes, thank you @7fELF